packages/flame_audio/example/lib/main.dart

```diff
@@ -39,9 +39,9 @@ class AudioGame extends FlameGame with TapDetector {
 
   Rect get button => Rect.fromLTWH(20, size.y - 300, size.x - 40, 200);
 
-  void startBgmMusic() {
-    FlameAudio.bgm.initialize();
-    FlameAudio.bgm.play('music/bg_music.ogg');
+  Future<void> startBgmMusic() async {
+    await FlameAudio.bgm.initialize();
+    await FlameAudio.bgm.play('music/bg_music.ogg');
```

Comment:
It also works withoud a Future, but it's cleaner and more future proof to await the initialize method.